clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -2374,10 +2504,20 @@ private ListOffsetsResponse listOffsetsResponse(Map<TopicPartition, Long> partit
     }
 
     private static class FetchInfo {
+        long logFirstOffset;
+        long logLastOffset;
```

Comment:
I considered inferring these from the sequence of other mock interactions, but it seemed more sensible to just have a way to specify them explicitly.